### PR TITLE
Some minor styling touch ups

### PR DESCRIPTION
**Pull request by zephraph**
Some light touch-up styles work. 

Adds palette for its auto-generated stylesheet. I'm using that to get the fonts closer. Also added the `number` library to fix the currency formatting.

image

### CODE REVIEW

File: mix.lock [hunk 1]
```diff
@@ -24,6 +24,7 @@
   "mimerl": {:hex, :mimerl, "1.2.0", "67e2d3f571088d5cfd3e550c383094b47159f3eee8ffa08e64106cdf5e981be3", [:rebar3], [], "hexpm"},
   "money": {:hex, :money, "1.4.0", "d9e1725832073566dc0c015603b1e82d69f5bd6c51299e9558e61aa6158044da", [:mix], [{:ecto, "~> 1.0 or ~> 2.0 or ~> 2.1 or ~> 3.0", [hex: :ecto, repo: "hexpm", optional: true]}, {:phoenix_html, "~> 2.0", [hex: :phoenix_html, repo: "hexpm", optional: true]}], "hexpm"},
   "neuron": {:hex, :neuron, "1.1.1", "17608cec9c9cdddad33e671d8c9061790414580c94963122ef209f48ed1ed897", [:mix], [{:httpoison, "~> 1.0", [hex: :httpoison, repo: "hexpm", optional: false]}, {:poison, "~> 3.1", [hex: :poison, repo: "hexpm", optional: false]}], "hexpm"},
+  "number": {:hex, :number, "1.0.0", "b88229d83830147186e8e3f133395fa9faf4f17126cdfeb3e2d19cc538d35ed5", [:mix], [{:decimal, "~> 1.5", [hex: :decimal, repo: "hexpm", optional: false]}], "hexpm"},
```
Comment (ashkan18) [inline mix.lock]: you may need to run `mix deps.get` which should remove this.

Comment (zephraph) [reply mix.lock]: Hmm.... I did. I just did it again and there's no change.

Comment (zephraph) [reply mix.lock]: Manually updated it for now.

**Comment by ashkan18** [comment]
merge on green

**Pull request merged**